### PR TITLE
Warn when the same predicate is used in more than one property

### PR DESCRIPTION
**Pull request by hectorcorrea**
This addresses issue #662

### CODE REVIEW

File: lib/active_fedora/attributes.rb [hunk 1]
```diff
@@ -171,16 +171,37 @@ def multiple?(field)
...
         ActiveTriples::Reflection.add_reflection self, name, reflection
       end
 
       private
 
+      def warn_duplicate_predicates new_name, new_properties
+        # TODO: Simplify logic once I am sure I got it right.
+        self.properties.each do |property, config| 
+          if config.predicate == new_properties[:predicate]
+            puts "- potential issue for predicate #{config.predicate} between #{new_name} and #{property}"
```
Comment (jcoyne) [inline lib/active_fedora/attributes.rb]: This should be `ActiveFedora::Base.logger`

File: lib/active_fedora/attributes.rb [hunk 2]
```diff
@@ -171,16 +171,37 @@ def multiple?(field)
...
+            if config.class_name == new_properties[:class_name]
+              if config.multiple? == new_properties[:multiple]
+                puts "\twe have a problem, warn the user"
+                # ActiveFedora::Base.logger.warn "Same predicate (#{config.predicate}) used for properties #{new_name} and #{property}"
+                ActiveFedora::Base.logger.warn "Same predicate"
+              else
+                puts "\tno real issue (different cardinality)"
```
Comment (jcoyne) [inline lib/active_fedora/attributes.rb]: I think this is one of the issues we're trying to avoid actually.

**Comment by hectorcorrea** [comment]
@jcoyne Updated to warn based on predicate alone (as per https://github.com/projecthydra/active_fedora/issues/662#issuecomment-69339032) Let me know what you think.

**Comment by jcoyne** [comment]
:+1:

**Pull request merged**